### DISCUSSION TITLE
Is Zigbee firmware for Sonoff Zigbeebridge stack able to do coordinator to cooridnator transfer ?

### DISCUSSION THREAD

**Discussion by gamer123** [arendst/Tasmota#10821] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/5868875/106806460-4bd7d500-6668-11eb-8a1f-ebc9817d5954.png)

The devices do not firm a network and i want to emulate the ECU coordinator and i would like to use a sonoff zigbee bridge to do so. 

Have to send and receive this :
Inverter ID ECU ID
ECU -->RX17    408000071212ffff10ffff 80971b01a3d8
ECU-->RX18     408000071212
Inverter-->RX21 ff1a408000071212
ECU--> RX23    408000071212 a3d8 10fff f80971b01a3d8
Inverter --> RX24 (sends 128Byte)
Inverter -->RX29 ffff408000071212
(ECU Broadcast -->RX30 80971b01a3d8 )

ECU --> 80971b01a3d8 FBFB06BB000000000000C1FEFE
Inerter sends around 121 Byte)

**Comment by ascillato** [comment, marked as answer]
Sorry, not supported.
